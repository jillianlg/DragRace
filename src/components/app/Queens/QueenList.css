ul{
  border: solid 1px pink;
  margin-bottom: .10vw;
  width: 75vw;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: center;
  align-items: center;
  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
  font-size: 14px;
  text-align: center;

  li{
    list-style-type: none;
  }

  img{
    padding: .6vw;
    margin: .6vw;
  }
}

