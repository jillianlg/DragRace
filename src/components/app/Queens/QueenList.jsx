import React from 'react';
import PropTypes from 'prop-types';

const QueenList = () => ();

QueenList.propTypes = {
  queens: propTypes.arrayOf(PropTypes.shape({
    image_url: PropTypes.string.isRequired,
    name: PropTypes.string.isRequired
  })).isRequired
};

export default QueenList;
