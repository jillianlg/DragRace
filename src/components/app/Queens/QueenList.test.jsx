import React from 'react';
import { render, cleanup } from '@testing-library/react';
import QueenList from './QueenList';


describe('QueenList component', () => {
  afterEach(() => cleanup());
  it('renders QueenList', () => {
    const { asFragment } = render(
      <QueenList
        queens={[]}
      />);
    expect(asFragment()).toMatchSnapshot();
  });
});
