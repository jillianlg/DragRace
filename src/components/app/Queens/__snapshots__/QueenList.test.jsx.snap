// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QueenList component renders QueenList 1`] = `
<DocumentFragment>
  <div
    class="QueenList"
  >
    <ul
      data-testid="queens"
    />
  </div>
</DocumentFragment>
`;
